app/views/applications/show.html.erb

```diff
@@ -0,0 +1,49 @@
+<h1><%= @application.name %></h1>
+<h3>Address: <%= @application.address %></h3>
+<h3><i>Status: <%= @application.status %></i></h3>
+<p><%= @application.description %></p>
+
+<% @pets.each do |pet| %>
+  <%= link_to "#{pet.name}", "/pets/#{pet.id}" %></br>
+<% end %>
+
+<% if @application.status == "In Progress" %>
+  <div id="add-pet">
+    <h3> Add a Pet to this Application </h3>
+    <%= form_with url: "/applications/#{@application.id}", method: :get, data: {turbo: false} do |form| %>
+      <%= form.label :search_name, "Enter a name: " %>
+      <%= form.text_field :search_name %>
+      <%= form.submit "Submit" %>
+    <% end %>
+  </div>
+<% end %>
+
+<% if @pets_search_name.present? %>
+  <% @pets_search_name.each do |pet| %>
+    <div id="pet_name">
+      <h3><%= pet.name %></h3>
+      <%= form_with url: "/applications/#{@application.id}", method: :get, data: {turbo: false} do |form| %>
+        <%= form.hidden_field :pet_to_adopt_id, value: "#{pet.id}" %>
+        <%= form.submit "Adopt this Pet" %>
+    <% end %>
+    </div>
+    <p> Age: <%= pet.age %></p>
+    <p> Breed: <%= pet.breed %></p>
+    <p> Adoptable: <%= pet.adoptable %></p>
+  <% end %>
+    <% elsif @pets_search_name.blank? && @pets_search_name != false %>
+      <p> No pets were found with that name </p>
+<% end %>
+
+<% if @wanted_pet.present? %>
+    <h3><%= @wanted_pet.name %></h3>
```

Comment:
so this just puts the name as text on the application, but we need to figure out a way to create a new application_pet record with the appropriate values (application_id and pet_id) when you click "Adopt This Pet" so that there's a link to the pet you've added and not just a header with the pet name